src/Connection/AbstractConnection.php

```diff
@@ -126,7 +126,7 @@ public function readResponse(CommandInterface $command)
      * @param string $message Error message.
      * @param int    $code    Error code.
      */
-    protected function onConnectionError($message, $code = null)
+    protected function onConnectionError($message, $code = 0)
```

Comment:
They didn't change anything, since the code is already changed in the exception class 

https://github.com/predis/predis/blob/v1.1/src/Connection/AbstractConnection.php#L146-L158